### PR TITLE
Replace posixpath.abspath/relpath to os.path.abspath/relpath

### PR DESCRIPTION
**Pull request by fhh2626**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 
 *.pyc
+BFEE2Gui.py
```
Comment (HanatoK) [inline .gitignore:3]: Why is `BFEE2Gui.py` added to `.gitignore`?

**Review by HanatoK — COMMENTED** [review]
Other changes looks good to me.

**Comment by HanatoK** [comment]
@fhh2626 Still, I am curious why the old code (before this commit) worked on my Windows 11 VM, but not on your machine. Any ideas?

**Comment by fhh2626** [comment]
> @fhh2626 Still, I am curious why the old code (before this commit) worked on my Windows 11 VM, but not on your machine. Any ideas?

No. Actually it works after some reinstall of BFEE2

**Pull request merged**